engines/saga/itedata.cpp

```diff
@@ -569,6 +615,13 @@ const char *whineStr[][NUM_WHINES] = {
 		"Sigh...",
 		"Credo che questa faccenda ti abbia dato alla testa, Sakka!",
 		"Beh, non ricordo regole specifiche contro i suggerimenti."
+	},
+	{
+		"Rohh, allez Sakka!",
+		"Un indice ne peut pas faire de mal.",
+		"Pfff...",
+		"Je crois que ton carnet te monte \205 la tête, Sakka!",
```

Comment:
Should that be 't\210te' rather than 'tête'?

Reply:
Yes you're right i forgot this one :( Thanks